R/IncidenceRate.R

```diff
@@ -0,0 +1,180 @@
+# @file IncidenceRate
+#
+# Copyright 2020 Observational Health Data Sciences and Informatics
+#
+# This file is part of ROhdsiWebApi
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#' Get definitions for incident rate analysis
+#'
+#' @details
+#' Returns an R-objects with definitions for incident rate analysis
+#'
+#' @param baseUrl            The base URL for the WebApi instance, for example:
+#'                           "http://server.org:80/WebAPI".
+#' @param incidenceRateId    The Atlas ID for incidence rate analysis.
+#' @return                   Get definitions for incident rate analysis
+#'
+#' @examples
+#' \dontrun{
+#' getIncidenceRateDefinitions <- function(baseUrl "http://server.org:80/WebAPI",
+#'                                          incidenceRateId = 296)
+#'                                          )
+#' }
+#' @export
+getIncidenceRateDefinitions <- function(baseUrl,
```

Comment:
The function name suggests plural, but the argument name (incidenceRateId) suggests singular. Why is this function returning multiple objects? (is it?) This would be different from say `getCohortDefinition`. 

I propose making it so this function returns a single incidence rate definition. If the user wants multiple, they can loop themselves. Unless there's a strong reason to stick to multiple? Are there multiple IR definitions with the same ID?